operator-common/src/main/java/io/strimzi/operator/common/model/ResourceType.java

```diff
@@ -2,36 +2,40 @@
  * Copyright 2018, Strimzi authors.
  * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
  */
-package io.strimzi.operator.cluster.model;
+package io.strimzi.operator.common.model;
 
 import io.strimzi.api.kafka.model.KafkaAssembly;
 import io.strimzi.api.kafka.model.KafkaConnectAssembly;
 import io.strimzi.api.kafka.model.KafkaConnectS2IAssembly;
+import io.strimzi.api.kafka.model.KafkaUser;
 
-public enum AssemblyType {
+public enum ResourceType {
```

Comment:
Why changing to `ResourceType` while we are using "assembly" in all the other parts of the code (i.e. in this declaration we are still using `KafkaAssembly` and not `KafkaResource`). I think that "assembly" provides the better idea that "Kafka", for example, is something made of more Kubernetes "resources".

Reply:
I do not think the KafkaUser is Assembly. It is just Resoruce. That is why I renamed it.

Reply:
On the KafkaUser I can agree but not on the others, otherwise we should change KafkaAssembly to KafkaResource or finding a better name.

Reply:
I'm not sure what others you mean.

Reply:
`KafkaAssembly` for example